### ISSUE TITLE
[bug] sublime text 3 - No parser and no filepath given

### ISSUE THREAD

**Issue by DoubleU23** [sheerun/prettier-standard#71]
TL;DR
the "SublimeJsPrettier" setup described in your README, doesn't work for me

## problem
__i tried to use "prettier-standard" with SublimeText3 package "JsPrettier"...__  

after struggling around with some unrecognized rule definitions (#72),
i __managed to successfully run "prettier-standard" via CLI__  

(__sideinfo: "prettier-eslint" was running fine in "JsPrettier"__)  
but when i change JsPrettier's config `prettier_cli_path` to "prettier-standard"...  
i always get weird errors... (permission denied)

tried several path combinations... here are the results:  

__prettier-eslint-cli__  

`"prettier_cli_path": "node_modules/prettier-eslint-cli/dist/index.js"`  - __RUNNING FINE__
`npx prettier-eslint-cli "./path/glob" - __running__ (but triggers npm ? install/update )

`"prettier_cli_path": "node_modules/prettier-eslint-cli"`  - __permission denied__  
`prettier-eslint-cli "./path/glob"` - __running fine__

__prettier-eslint__  
`"prettier_cli_path": "node_modules/prettier-eslint/dist/index.js"` - __Exec format error__  
`npx prettier-eslint "./path/glob" - __running fine__

`"prettier_cli_path": "node_modules/prettier-eslint"` - __permission denied__

__prettier-standard__  
`"prettier_cli_path": "node_modules/prettier-standard"`   - __permission denied__  
`npx prettier-standard "./path/glob" - __running fine__

`"prettier_cli_path": "node_modules/prettier-standard/src/index.js"` - __"No parser and no filepath given"__  

tried to use "prettier-standard" globally... wihtout success

## detailed errors  
<details>
<summary>Permission denied</summary>

```
error: JsPrettier - [Errno 13] Permission denied
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 1072, in run_
    return self.run(edit)
  File "/home/doubleu23/.config/sublime-text-3/Packages/JsPrettier/JsPrettier.py", line 283, in run
    provide_cursor=True)
  File "/home/doubleu23/.config/sublime-text-3/Packages/JsPrettier/JsPrettier.py", line 378, in format_code
    shell=is_windows())
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1448, in _execute_child
PermissionError: [Errno 13] Permission denied
```
</details>

<details>
<summary>"No parser and no filepath given"</summary>

```
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 1072, in run_
    return self.run(edit)
  File "/home/doubleu23/.config/sublime-text-3/Packages/JsPrettier/JsPrettier.py", line 283, in run
    provide_cursor=True)
  File "/home/doubleu23/.config/sublime-text-3/Packages/JsPrettier/JsPrettier.py", line 406, in format_code
    return stdout.decode('utf-8'), int(new_cursor)
ValueError: invalid literal for int() with base 10: "No parser and no filepath given, using 'babylon' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred."
```
</details>

<details>
<summary>"Exec format Error"</summary>

```
error: JsPrettier - [Errno 8] Exec format error
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 1072, in run_
    return self.run(edit)
  File "/home/doubleu23/.config/sublime-text-3/Packages/JsPrettier/JsPrettier.py", line 283, in run
    provide_cursor=True)
  File "/home/doubleu23/.config/sublime-text-3/Packages/JsPrettier/JsPrettier.py", line 378, in format_code
    shell=is_windows())
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1448, in _execute_child
OSError: [Errno 8] Exec format error
```
</details>

## chmod info  
i double checked the files and they are (testwise) on __`chmod 777`!__  
as the Error points out "prettier-standard" isn't executable (python - _execute_child)  
i also set 777 on the node_module/prettier-standard...

__any idea(s)__?

**Comment by sheerun** [comment]
prettier-standard 9 no longer uses eslint for formatting so it should fix this issue
